concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-17	17	attraktiv	attractive	2410	GOOD-LOOKING	13.839	6.667	6.25	5.583	6.333	6.458	5.958	6.208	2.75	3.417	3.5	4.25	3.083	3.875	3.479	4.667	4.75	4.333	4.75	4.708	4.542	4.625	6.636	6.417	6.083	6.167	6.522	6.125	6.319	5.417	5.25	4.917	3.75	5.333	4.333	4.833	2.083	1.583	2.25	1.917	1.833	2.083	1.958	6.438	5.813	1.375
+Gruehn-2008-200-18	18	aufbrausend	quick-tempered			376	2.917	2.75	1.667	2.083	2.833	1.875	2.354	6.167	5.75	6.583	6.5	5.958	6.542	6.25	2.917	3.25	3.75	2.25	3.083	3	3.042	4.833	5.833	4.667	4.917	5.333	4.792	5.063	3.5	3.5	2.917	2.5	3.5	2.708	3.104	2.25	2.25	2.417	2	2.25	2.208	2.229	2.688	2.375	1.083
+Gruehn-2008-200-19	19	aufgewuehlt	agitated			1.217	3.25	3.167	3	3.167	3.208	3.083	3.146	6	5.667	6.083	6.417	5.833	6.25	6.042	2.833	2.667	4.167	4.167	2.75	4.167	3.458	4	4.583	5.083	4.417	4.292	4.75	4.521	3.583	3.583	3.583	3.25	3.583	3.417	3.5	2.417	1.917	2.167	2.083	2.167	2.125	2.146	3.25	3.375	2
+Gruehn-2008-200-20	20	aufmerksam	attentive	1908	ATTENTIVE	27.967	5.667	5.583	6.25	5.833	5.625	6.042	5.833	4.083	4.333	5.083	5.636	4.208	5.348	4.766	6	6	5.083	5.833	6	5.458	5.729	3.833	4.583	5.167	5	4.208	5.083	4.646	6.167	5.667	5.833	5.833	5.917	5.833	5.875	3.083	2.833	2.75	2.667	2.958	2.708	2.833	6.25	6.188	1.583
+Gruehn-2008-200-21	21	ausdauernd	persistent			1.174	5.417	5.75	6.083	6.25	5.583	6.167	5.875	4.417	4.25	3.333	4.167	4.333	3.75	4.042	5.583	6.167	5.5	5.333	5.875	5.417	5.646	4	3.667	4.25	4.25	3.833	4.25	4.042	5.25	5.083	6.083	5.5	5.167	5.792	5.479	2.75	2.083	3.25	2.583	2.417	2.917	2.667	6.25	5.625	2
+Gruehn-2008-200-22	22	ausgeglichen	balanced			11.467	6.75	6.25	6.417	6.083	6.5	6.25	6.375	1.667	2.417	2.417	2.5	2.042	2.458	2.25	5.667	5	4.75	5.083	5.333	4.917	5.125	3.75	3.5	4.5	2.833	3.625	3.667	3.646	4.917	4.083	5.333	5	4.5	5.167	4.833	3.333	4.083	3.833	4.083	3.708	3.958	3.833	6.813	6.375	1.833
+Gruehn-2008-200-23	23	autoritaer	authoritarian			1.537	2.75	2.167	2.167	2.083	2.458	2.125	2.292	5.167	5.75	5.5	5.917	5.458	5.708	5.583	5.5	5.917	5.167	4.667	5.708	4.917	5.313	4.75	5.75	4.333	5.417	5.25	4.875	5.063	3.25	2.917	4.083	2.917	3.083	3.5	3.292	3.917	3.5	3.818	3.25	3.708	3.522	3.617	3	1.875	1
+Gruehn-2008-200-24	24	bedaechtig	deliberate			1.866	4.25	3.833	4.917	4.25	4.042	4.583	4.313	2.917	2.917	2.5	3.083	2.917	2.792	2.854	5	5	4.167	3.833	4.1	4	4.5	3.583	3.417	4.5	3.5	3.5	4	3.75	3.917	4	4.5	4.667	3.958	4.583	4.271	4	4.167	3.917	4	4.083	3.958	4.021	5	5.438	1.833
+Gruehn-2008-200-25	25	begeistert	enthusiastic	3972	EXCITED	24.183	6.083	6	6	5.75	6.042	5.875	5.958	4.5	4.083	4.75	4.583	4.292	4.667	4.479	4.833	4.083	3.833	4.583	4.458	4.208	4.333	5.455	6	5.667	5.667	5.739	5.667	5.702	5.667	5	5.917	4.917	5.333	5.417	5.375	2.5	1.917	1.833	2	2.208	1.917	2.063	6.438	6.063	1.708
```

Comment:
I think this mapping works, but checking the concept itself I'm not sure whether I'm happy with the definition. Will think about that a bit more. What do the others think?

Reply:
Yes, I was also a bit surprised about the definition. It might be better to unmap and update the definition of the concept. @MiraAhmedovic Could you please add an issue linking to @chrzyki's comment above so that we can update the definition later.

Reply:
See #1436.